### PR TITLE
chore: add CODEOWNERS file

### PR DESCRIPTION
**Pull request by Akuukis**
## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [ ] Ensure the tests pass
4. [ ] Run `./python-format.sh` to make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/main/CONTRIBUTING.md)
5. [x] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes
8. [ ] Create a testing plan for the reviewer and add it to the Testing section
9. [ ] Add frontend or backend tag and any other appropriate tags to this pull request

## Description

Internal process configuration with no changes to the product.

## Notes

Let's declare ownership of code. Please comment or directly push to this PR to add yourself (or others) to CODEOWNERS before approving. [Read more](https://www.notion.so/kobotoolbox/KIP-Pull-Requests-1147e515f6548094bf2ec594ed1f5617#1287e515f65480e08069e274595d8333) in internal docs. In short:
- add yourself if you or others consider you a subject expert of that code, or
- add yourself if you want to always receive review requests on changes of that code

## Testing

Not testable. After merging, should suggest reviewers based on the CODEOWNERS file according to [github docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

### CODE REVIEW

**Review by rgraber — APPROVED** [review]
Added myself for audit logs. LGTM

**Review by pauloamorimbr — APPROVED** [review]
LGTM!

**Review by magicznyleszek — APPROVED** [review]

**Comment by magicznyleszek** [comment]
Looks good. I will wait for everyone to take what they want, and I will have the rest 🤑

**Review by RuthShryock — APPROVED** [review]

**Review by noliveleger — COMMENTED** [review]
@jnm, @bufke , @JacquelineMorrissette please review [my changes](https://github.com/kobotoolbox/kpi/pull/5213/commits/7a2349581c3bc2f70db894d4f51cd74236bce13e)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+
+#### Frontend section.
+
+# Default owner
+.storybook/*                        @magicznyleszek
+cypress/*                           @magicznyleszek
+jsapp/*                             @magicznyleszek
+patches/*                           @magicznyleszek
```
Comment (magicznyleszek) [inline .github/CODEOWNERS:32]: @p2edwards Wold you like to take ownership of patches?

**Review by bufke — APPROVED** [review]

**Comment by jnm** [comment]
> @jnm, @bufke , @JacquelineMorrissette please review [my changes](https://github.com/kobotoolbox/kpi/pull/5213/commits/7a2349581c3bc2f70db894d4f51cd74236bce13e)

@noliveleger, I have contradicted a few of them. You can see my rationale in my commit messages. LMK if you want to discuss anything

**Review by jnm — APPROVED** [review]

**Comment by Akuukis** [comment]
Sorted, grouped and clarified that last **line** wins. Added the missing github folder, set @p2edwards as owner for patches folder as per his :+1: above.

File: .github/CODEOWNERS (hunk 1 above)
Comment (Akuukis) [reply .github/CODEOWNERS]: done

**Review by Guitlle — APPROVED** [review]

**Pull request merged**